ppr-api/src/ppr_api/models/registration.py

```diff
@@ -12,37 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """This module holds common statement registration data."""
+
+# pylint: disable=too-many-statements, too-many-branches
+
 #import pycountry
 
 from enum import Enum
 from http import HTTPStatus
+from datetime import date
```

Comment:
in the other apps we're mostly importing our util/datetime.py which wraps datetime and adds in non-naive now()

Reply:
It's only used for date.fromisoformat...is this a problem?

Reply:
To clarify, date.fromisoformat is used when creating a financing statement and creating a renewal statement. The code  can be corrected when developing those transactions.